testsuite/features/step_definitions/common_steps.rb

```diff
@@ -713,14 +713,16 @@
 end
 
 When(/^I reboot the "([^"]*)" minion through the web UI$/) do |host|
-  step %(Given I am on the Systems overview page of this "#{host}")
-  step %(When I follow first "Schedule System Reboot")
-  step %(Then I should see a "System Reboot Confirmation" text")
-  step %(And I should see a "Reboot system" button")
-  step %(When I click on "Reboot system")
-  step %(Then I should see a "Reboot scheduled for system" text")
-  step %(And I wait at most 600 seconds until event "System reboot scheduled by admin" is completed")
-  step %(Then I should see a "This action's status is: Completed" text")
+  steps %(
+    Given I am on the Systems overview page of this "#{host}"
+    When I follow first "Schedule System Reboot"
+    Then I should see a "System Reboot Confirmation" text
+    And I should see a "Reboot system" button
+    When I click on "Reboot system"
+    Then I should see a "Reboot scheduled for system" text
+    And I wait at most 600 seconds until event "System reboot scheduled by admin" is completed
+    Then I should see a "This action's status is: Completed" text
+         )
```

Comment:
I can't see the difference. Could you explain the fix please?

Reply:
This was the error we had in BV after the previous PR was merged:
```
Undefined dynamic step: "Given I am on the Systems overview page of this "sle15sp2_minion"" (Cucumber::UndefinedDynamicStep)
./features/step_definitions/common_steps.rb:550:in `/^I reboot the "([^"]*)" minion through the web UI$/'
features/build_validation/smoke_tests/sle15sp2_minion_smoke_tests.feature:157:in `I reboot the "sle15sp2_minion" minion through the web UI'
```
and it's the multiple steps definition that was not correct. Instead of one line per step, we wanted multiple steps in one. It's a syntax thing basically that with multiple lines with `step` it was like defining multiple possibilities of definition in one step which was ambiguous, instead of using `steps` and use already defined steps to wrap around

Reply:
Dominik now tested the fix for this, the current PR directly in the BV env and it fixes the issue.

Reply:
Yes, as Kostas already explained, the old solution had a `step %()` in every line which is apparently not possible in Cucumber.